crates/sui-json-rpc/src/read_api.rs

```diff
@@ -143,6 +148,25 @@ impl ReadApiServer for ReadApi {
             .cloned()
             .map_or(cursor, |o_info| Some(o_info.object_id));
```

Comment:
On 138 we should use this https://github.com/MystenLabs/sui/blob/99f6d39e680b0f1878ddeab206d1d83792d95eb3/crates/sui-json-rpc/src/coin_api.rs#L121-L131